offchain/authority-claimer/src/mock.rs

```diff
@@ -0,0 +1,75 @@
+// (c) Cartesi and individual authors (see AUTHORS)
```

Comment:
This file is specific to the broker part, so I would call it `broker_mock.rs` instead.

Reply:
Done at 9ecb93ba